spanner/google/cloud/spanner/client.py

```diff
@@ -104,21 +103,16 @@ class Client(_ClientFactoryMixin, _ClientProjectMixin):
     _database_admin_api = None
     _SET_PROJECT = True  # Used by from_service_account_json()
 
+    SCOPE = (SPANNER_ADMIN_SCOPE,)
```

Comment:
Worth noting this is a change to the public interface of `spanner.Client` (previously `SCOPE` was the [default][1] value)

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/2083542f3d799a5011fcd4d3092257b958d5511a/core/google/cloud/client.py#L126